File: Examples/Features/Comments/ConfigCommentLayout.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Comments
{
    public class ConfigCommentLayout : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            IComment commentC3 = worksheet.Range["C3"].AddComment("Range C3's comment.");
            commentC3.Shape.Line.Color.RGB = Color.LightGreen;
            commentC3.Shape.Line.Weight = 3;
            commentC3.Shape.Line.Style = GrapeCity.Documents.Excel.Drawing.LineStyle.ThickThin;
            commentC3.Shape.Line.DashStyle = GrapeCity.Documents.Excel.Drawing.LineDashStyle.Solid;
            commentC3.Shape.Fill.Color.RGB = Color.Pink;
            commentC3.Shape.Width = 100;
            commentC3.Shape.Height = 200;
            commentC3.Shape.TextFrame.TextRange.Font.Bold = true;
            commentC3.Visible = true;
        }
    }
}

File: Examples.VB/Features/Comments/ConfigCommentLayout.vb
Imports System.Drawing

Namespace Features.Comments
    Public Class ConfigCommentLayout
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            Dim commentC3 As IComment = worksheet.Range!C3.AddComment("Range C3's comment.")

            commentC3.Shape.Line.Color.RGB = Color.LightGreen
            commentC3.Shape.Line.Weight = 3
            commentC3.Shape.Line.Style = LineStyle.ThickThin
            commentC3.Shape.Line.DashStyle = LineDashStyle.Solid
            commentC3.Shape.Fill.Color.RGB = Color.Pink
            commentC3.Shape.Width = 100
            commentC3.Shape.Height = 200
            commentC3.Shape.TextFrame.TextRange.Font.Bold = True
            commentC3.Visible = True
        End Sub
    End Class
End Namespace
